src/app/modals/about-modal-config.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * A config containing properties for About Modal
+ */
+export class AboutModalConfig {
+  /**
+   * Text explaining the version or copyright
+   */
+  additionalInfo?: string;
+
+  /**
+   * Product copyright information
+   */
+  copyright?: string;
+
+  /**
+   * The alt text for the corner graphic
+   */
+  imgAlt?: string;
```

Comment:
Please rename to imageAlt. The other components are using 'image' instead of 'img'.